gravitee-am-identityprovider/gravitee-am-identityprovider-mongo/src/test/java/io/gravitee/am/identityprovider/mongo/authentication/MongoAuthenticationProviderTest.java

```diff
@@ -41,6 +41,13 @@
     @Autowired
     private AuthenticationProvider authenticationProvider;
 
+    /**
+     * This is a workaround for Macs with M1 ships
+     * Link to the github issue:
+     * https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/337#issuecomment-794303570
+     * */
+    static { System.setProperty("os.arch", "i686_64"); }
+
```

Comment:
Should this be really here ?
Isn't it something we should just mention in the concerned maven plugin ?
```
-Dos.arch= i686_64
```

Reply:
Indeed, maybe we can do it at mvn level.

Reply:
yes we can, I tried it and it works fine in mvn level when calling "mvn clean install" just add "-Dos.arch=i686_64"